sample_size/scripts/sample_size_run.py

```diff
@@ -0,0 +1,38 @@
+def main() -> None:
+    """
+    Calculate sample size based on user inputs for
+        1. metric type: Boolean, Numeric, or Ratio (case insensitive)
+        2. power analysis parameters: alpha, power, and mde
+        3. inputs specific to metric type:
+            * Boolean: probability
+            * Numeric: variance
+            * Ratio: mean and variance of numerator and denominator and their covariance
+
+    NOTES:
+        1. default statistical power is used in this script all the time
+        2. the calculator supports single metric per calculator for now
+    """
+    from sample_size.sample_size_calculator import SampleSizeCalculator
+    from sample_size.scripts.utils import get_alpha
+    from sample_size.scripts.utils import get_metric_metadata_from_input
+    from sample_size.scripts.utils import register_metric
+
+    try:
+        # Get alpha for power analysis
+        alpha = get_alpha()
```

Comment:
Should we also be collecting power here?

Reply:
FYI an existing thread about this (already resolved so i can only paste screenshot here)
![image](https://user-images.githubusercontent.com/46336255/145322666-72b9b2ae-ee78-47d2-840e-84148ca532b4.png)

Reply:
Ope, sorry I missed that!